scripts/inline_comments_data_collection.py

```diff
@@ -47,16 +47,43 @@ def find_revision_from_patch(patch_id):
     return revision_phid
```

Comment:
We should also be able to avoid the `api.search_diffs` calls

Reply:
Not 100% sure if the revisions object contains all the necessary information to replace `api.search_diffs()`, but will look into it!

Reply:
You're using it to get the `revisionPHID`, which should be in the revision object (maybe called just PHID?)